client/packages/system/src/Patient/Encounter/CreateEncounterModal.tsx

```diff
@@ -292,6 +296,7 @@ export const CreateEncounterModal: FC = () => {
                           highlightedDays: getHighlightedDays(),
                         } as any,
                       }}
+                      showTime
```

Comment:
This doesn't work... onChange sets it back to now 😅  Would just remove? Haven't heard request to be able to set the encounter time (though not sure why not lol)

Reply:
O_O It worked for me... I set time... got redirected and Encounter startdatetime was set but will double check

Reply:
[d01ca5c](https://github.com/msupply-foundation/open-msupply/pull/6198/commits/d01ca5c764223410058a7b669d559f8c999dfd78)